nature/migrations/0001_initial.py

```diff
@@ -694,7 +694,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='transaction',
             name='transaction_type',
-            field=models.ForeignKey(db_column='tapahtumatyyppiid', on_delete=django.db.models.deletion.PROTECT, to='nature.TransactionType'),
+            field=models.ForeignKey(related_name='transactions', db_column='tapahtumatyyppiid', on_delete=django.db.models.deletion.PROTECT, to='nature.TransactionType'),
```

Comment:
I think we also need to change `007_add_verbose_name` for this `related_name`?

Reply:
Fixed

Reply:
https://github.com/City-of-Helsinki/ltj/pull/111/commits/d07aad0eac1e50a4e266f04c422bcbb43ad43356#diff-c0899afc2931712cdec10aa506537df2L1130